cross-platform/react-app/src/index.tsx

```diff
@@ -5,11 +5,12 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
-import App from "./frontend/App";
+import { App, CameraSampleApp } from "./frontend/App";
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <CameraSampleApp />
```

Comment:
I assume you already know, but a switch statement above could read a variable that gets set via the hash params, then default to App, but use CameraSampleApp if the variable is set to "CameraSample". Something like:

```ts
let app;
switch (window.itmSampleName) {
  case "CameraSample":
    app = CameraSampleApp;
    break;
  default:
    app = App;
    break;
}
```

Reply:
I enhanced index.tsx to dynamically import either App or CameraSampleApp depending on an env var. You can see from package.json examples of how it is passed in.